diff --git a/src/intel/vulkan/grl/meson.build b/src/intel/vulkan/grl/meson.build
index 61cb7aa..dfa53e8 100644
--- a/src/intel/vulkan/grl/meson.build
+++ b/src/intel/vulkan/grl/meson.build
@@ -138,6 +138,7 @@ foreach t : [['125', 'gfx125', 'dg2']]
         '-DMAX_HW_SIMD_WIDTH=16', '-DMAX_WORKGROUP_SIZE=16',
         '-I' + join_paths(meson.current_source_dir(), 'gpu'),
         '-I' + join_paths(meson.current_source_dir(), 'include'),
+        '-I/usr/lib64/clang/16/include',
       ],
       env: ['MESA_SHADER_CACHE_DISABLE=true',
             'MESA_SPIRV_LOG_LEVEL=error'],
