diff --git a/src/intel/shaders/meson.build b/src/intel/shaders/meson.build
index e8e2c31..4474d07 100644
--- a/src/intel/shaders/meson.build
+++ b/src/intel/shaders/meson.build
@@ -33,13 +33,13 @@ endforeach
 
 intel_shaders_clc_wa_args = []
 if with_intel_clc
-  if chosen_llvm_version_major >= 17
-    intel_shaders_clc_wa_args += ['--llvm17-wa']
+  if chosen_llvm_version_major >= 16
+    intel_shaders_clc_wa_args += ['--llvm16-wa']
   endif
 else
   _intel_clc_llvm_version = run_command(prog_intel_clc, '-M')
-  if _intel_clc_llvm_version.stdout().strip().version_compare('>= 17.0')
-    intel_shaders_clc_wa_args += ['--llvm17-wa']
+  if _intel_clc_llvm_version.stdout().strip().version_compare('>= 16.0')
+    intel_shaders_clc_wa_args += ['--llvm16-wa']
   endif
 endif
 
